src/documentation/programming-guide.md

```diff
@@ -1460,9 +1460,16 @@ An example might be if your pipeline reads log records from an input file, and e
 
 > **NOTE:** This content applies only to the Beam SDK for Java. The Beam SDK for Python does not support triggers.
 
-When collecting and grouping data into windows, Beam must determine when to emit the aggregated results of each window. If you use Beam's default windowing configuration and [default trigger](#default-trigger), Beam outputs the aggregated result when it [estimates all data has arrived](#watermarks-late-data), and discards all subsequent data for that window.
+When collecting and grouping data into windows, Beam uses **triggers** to determine when to emit the aggregated results of each window (referred to as a *pane*). If you use Beam's default windowing configuration and [default trigger](#default-trigger), Beam outputs the aggregated result when it [estimates all data has arrived](#watermarks-late-data), and discards all subsequent data for that window.
 
-**Triggers** allow you to change this default behavior and specify when Beam emits the aggregated results for each window (referred to as a *pane*). At a high level, triggers provide two additional capabilities compared to simply outputting at the end of a window:
+You can set triggers for your `PCollection`s to change this default behavior and specify when to emit the aggregated results for each window. Beam provides a number of pre-built triggers that you can set:
```

Comment:
"and specify when to emit" may be redundant with the same statement in the paragraph above, but seems fine either way.